Library/Homebrew/Gemfile

```diff
@@ -9,6 +9,7 @@ source "https://rubygems.org"
 gem "bootsnap", require: false
 gem "byebug", require: false
 gem "json_schemer", require: false
+gem "mdl", require: false
```

Comment:
This might make more sense to be in `docs/Gemfile`, and then it can be added to `rake test` like `html-proofer` is.

Reply:
Agreed. We should also actually run `rake test` for the docs folder when they change (perhaps with a new action).